devito/interfaces.py

```diff
@@ -189,6 +197,7 @@ def __init__(self, *args, **kwargs):
             self.shape = (time_dim,) + self.shape
             # Store final instance in symbol cache
             self._cache_put(self)
+        MemmapManager.setup(self, *args, **kwargs)
```

Comment:
Hi, could you please explain the cached functionality? If we create a data from symbol cache, it it just another symbol representing the same data that has already been created? If yes, then I agree, this is redundant. If not, we are I creating a all new data, then we cannot be sure it will go into the else branch.